### PR TITLE
LegalName: Adapt placeholders to account types

### PR DESCRIPTION
**Pull request by Betree**
Resolve https://github.com/opencollective/opencollective/issues/4653

### CODE REVIEW

File: components/CreateOrganizationForm.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ const orgMessages = defineMessages({
 
 const placeholders = {
   name: { id: 'placeholder.name', defaultMessage: 'e.g. Salesforce, Airbnb' },
-  legalName: { id: 'placeholder.legalName', defaultMessage: 'e.g. Salesforce.com, Inc., Airbnb, Inc.' },
+  examples: { id: 'examples', defaultMessage: 'e.g. {examples}' },
```
Comment (SudharakaP) [inline components/CreateOrganizationForm.js:67]:
```suggestion
  examples: { id: 'examples', defaultMessage: 'e.g., {examples}' },
```

File: components/CreateOrganizationForm.js [hunk 2]
```diff
@@ -229,7 +229,9 @@ const CreateOrganizationForm = props => {
                           <Field
                             as={StyledInput}
                             {...inputProps}
-                            placeholder={intl.formatMessage(placeholders.legalName)}
+                            placeholder={intl.formatMessage(placeholders.examples, {
+                              examples: 'Salesforce.com, Inc., Airbnb, Inc.',
```
Comment (SudharakaP) [inline components/CreateOrganizationForm.js:233]:
```suggestion
                              examples: 'Salesforce, Inc., Airbnb, Inc.',
```

File: components/edit-collective/Form.js [hunk 3]
```diff
@@ -109,6 +109,10 @@ class EditCollectiveForm extends React.Component {
         defaultMessage:
           'The legal name is private and shared with the hosts for donation receipts, tax forms and when you submit and expense. This name is not displayed publicly and it must be your legal name.',
       },
+      examples: {
+        id: 'examples',
+        defaultMessage: 'e.g. {examples}',
```
Comment (SudharakaP) [inline components/edit-collective/Form.js:114]:
```suggestion
        defaultMessage: 'e.g., {examples}',
```

**Review by SudharakaP — APPROVED** [review]
Thanks for making this change; there are some small things I noticed; 

1) `e.g.` should be `e.g.,` I think. See https://www.merriam-webster.com/dictionary/e.g. and examples there.

2) `Salesforce.com, Inc.` should be `Salesforce, Inc.`; refer https://en.wikipedia.org/wiki/Salesforce

File: components/edit-collective/Form.js (hunk 3 above)
Comment (Betree) [reply components/edit-collective/Form.js:114]: We have 14 places using `e.g.` right now, and none using `e.g,` - including places where that Alanna updated. I'm gonna strive for consistency for now and ask for feedback on Slack. We'll update everything if needed.

**Pull request merged**